src/runtime/virtcontainers/network.go

```diff
@@ -196,7 +196,17 @@ type Network struct {
 	NetmonPID int
 }
 
-func NewNetwork(config *NetworkConfig) (*Network, error) {
+func NewNetwork(configs ...*NetworkConfig) (*Network, error) {
+	if len(configs) > 1 {
```

Comment:
Can you clarify this part?

Reply:
I added a comment in the `network_linux.go` file for clarification:

```
// NewNetwork creates a new Linux Network from a NetworkConfig.
// The constructor is overloaded as it can be called with 0 or 1
// argument. The former is used to create empty networks, mostly
// for unit testing. Passing more than one NetworkConfig pointer
// will make the constructor fail.
```